src/stylesheets/variables.scss

```diff
@@ -28,6 +28,9 @@ $goal-content-height: 160px;
 $goal-header-background: #135ef2;
 $goal-header-background-light: #5295ed;
 
+$stat-background: #135ef2;
```

Comment:
Nit: duplication of the same background colours for goals and stats. This could be refactored into a single variable for better maintainability in the future.